File: Origami.plugin/Contents/Resources/Code Export/SpringSetup.js
// %@ transition

var %@Spring = springSystem
  .createSpringWithBouncinessAndSpeed(%@, %@)
  .addListener({
    onSpringUpdate: function(spring) {
      set%@Progress(spring.getCurrentValue());
    }
  });

var %@ = function(on) {
  %@Spring.setEndValue(on ? 1 : 0);
};

File: Origami.plugin/Contents/Resources/Code Export/SpringSetup.java
    %@Spring = springSystem.createSpring()
        .setSpringConfig(SpringConfig.withBouncinessAndSpeed(%@, %@))
        .addListener(new SimpleSpringListener() {
          @Override
          public void onSpringUpdate(Spring spring) {
            set%@Progress((float) spring.getCurrentValue());
          }
        });